<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="settings" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Algorithms</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="bubblesort">
               <property name="text">
                <string>Bubble Sort</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="selectionsort">
               <property name="text">
                <string>Selection Sort</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="insertionsort">
               <property name="text">
                <string>Insertion Sort</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="heapsort">
               <property name="text">
                <string>Heap Sort</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="mergesort">
               <property name="text">
                <string>Merge Sort</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="quicksort">
               <property name="text">
                <string>Quick Sort</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="start_pushButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="charts_t" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="charts">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>570</width>
               <height>523</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>8</number>
              </property>
              <property name="rightMargin">
               <number>8</number>
              </property>
              <property name="bottomMargin">
               <number>8</number>
              </property>
              <item>
               <widget class="QLabel" name="notice_label">
                <property name="text">
                 <string>Select at least one algorithm.</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>800</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>390</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
